packages/suite/src/components/suite/modals/ReduxModal/UserContextModal/UnstakeModal/UnstakeEthForm/UnstakeEthForm.tsx

```diff
@@ -16,9 +16,8 @@ const GreyP = styled(Paragraph)`
 const DividerWrapper = styled.div`
```

Comment:
Not sure if related but missing gap under warning 
![Screenshot 2024-03-13 at 13 10 48](https://github.com/trezor/trezor-suite/assets/33235762/9b282d86-3824-405d-a2dd-e6440a3c6f41)

Reply:
Done. https://github.com/trezor/trezor-suite/pull/11543/commits/7295f09ab08677797bacb5c857312e106242001d